presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestFixedDoubleBreakdownHistogram.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.fixedhistogram;
```

Comment:
The package name is inconsistent with the dir path. Either remove `fixedhistogram` from the package or add `fixedhistogram` to the directory

Reply:
A copy of this file existed in the directory above, but I removed it several `commit --amend` agos:

```
atavory-mbp:presto-main atavory$ find . -name 'FixedDoubleHistogram.java'
./src/main/java/com/facebook/presto/operator/aggregation/fixedhistogram/FixedDoubleHistogram.java
```

I hope we're looking at the same sources (if not, I must be making some mistake).

Reply:
`FixedDoubleHistogram` is fixed; the corresponding test is not... `TestFixedDoubleBreakdownHistogram` and `TestFixedDoubleHistogram` are in the wrong directory.